pyverbs/providers/efa/efadv.pyx

```diff
@@ -0,0 +1,102 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All rights reserved.
+
+cimport pyverbs.providers.efa.efadv_enums as dve
+cimport pyverbs.providers.efa.libefa as dv
+
+import logging
```

Comment:
you're not using logging and most probably won't be using PyverbsRDMAErrno (see the comment below)

Reply:
Done